doc: upadte migrate diff command (#3329)
ent/ent: doc/md/getting-started.mdx

    }
    ```

## Schema Migration

> Ent provides two approaches for running schema migrations: [Automatic Migration](/docs/migrate) and

Reviewer: ```suggestion
Ent provides two approaches for running schema migrations: [Automatic Migrations](/docs/migrate) and
```

> [Versioned migration](/docs/versioned-migrations). Here is a brief overview of each approach:

Reviewer: ```suggestion
[Versioned migrations](/docs/versioned-migrations). Here is a brief overview of each approach:
```

> ### Automatic Migration

Reviewer: ```suggestion
### Automatic Migrations
```

> With Automatic Migration, users can use the following API to keep the database schema aligned with the schema objects

Reviewer: ```suggestion
With Automatic Migrations, users can use the following API to keep the database schema aligned with the schema objects
```

defined in the generated SQL schema `ent/migrate/schema.go`:
```go
if err := client.Schema.Create(ctx); err != nil {
    log.Fatalf("failed creating schema resources: %v", err)
}
```

> This approach is mostly useful for prototyping, development, or testing. However, it is recommended to use the

Reviewer: ```suggestion
This approach is mostly useful for prototyping, development, or testing. Therefore, it is recommended to use the
```

> _Versioned Migration_ approach for mission-critical production environments. Using versioned migrations users know

Reviewer: ```suggestion
_Versioned Migration_ approach for mission-critical production environments. By using versioned migrations, users know
```

> beforehand what changes are being applied to their database, and can easily tune them for their needs.

Reviewer: ```suggestion
beforehand what changes are being applied to their database, and can easily tune them depending on their needs.
```

Read more about this approach in the [Automatic Migration](/docs/migrate) documentation.

> ### Versioned Migration

Reviewer: ```suggestion
### Versioned Migrations
```

> Unlike _Automatic Migration_, the _Version Migrations_ approach uses Atlas to automatically generate a set of migration

Reviewer: ```suggestion
Unlike _Automatic Migrations_, the _Version Migrations_ approach uses Atlas to automatically generate a set of migration
```
